SConstruct

```diff
@@ -378,6 +380,34 @@ elif env['COMPILER'] == 'msvc':
         env.Append(CCFLAGS=Split('/Od /Zi /MD'))
         env.Append(LINKFLAGS=Split('/DEBUG'))
 
+if not env['SHOW_CMDS']:
```

Comment:
I am not sure about the condition here, should that be `if env['SHOW_CMDS']` ?

Reply:
I copied the logic from arnold-core, where it's meant to be something like 'if SHOW_CMDS is True, just print the original verbose compile commands, otherwise show the minimal commands'